crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -415,6 +430,50 @@ impl BuildSpecCommand {
 			true
 		};
 
+		// Prompt the user for deterministic build only if the profile is Production.
+		let deterministic = deterministic || (profile == Profile::Production && cli
+			.confirm("Would you like to build the runtime deterministically? (Recommended for production builds)")
```

Comment:
```suggestion
			.confirm("Would you like to build the runtime deterministically? This requires a containerization solution (Docker/Podman) and is recommended for production builds.")
```

Can be iterated on to simplify

Reply:
Improved prompt to the user: https://github.com/r0gue-io/pop-cli/pull/425/commits/bb3bd7c1fe2ccfafd5a7ad3453746d2546fa0b52